src/exercise/obj/Debug/netcoreapp3.1/exercise.AssemblyInfoInputs.cache

```diff
@@ -0,0 +1 @@
+225760618b49a911e02cf82d367cf755d361b9c9
```

Comment:
cache files should be removed from git. Add following line to `.gitignore` -file (create if not exists):

```
*.cache
```

Also run `git rm src/exercise/obj/Debug/netcoreapp3.1/*.cache` and
`git rm src/exercise/obj/Debug/netcoreapp3.1/exercise.csprojAssemblyReference.cache` in git shell